src/main/java/nextstep/ladder/controller/Main.java

```diff
@@ -0,0 +1,26 @@
+package nextstep.ladder.controller;
+
+import java.util.List;
+
+import nextstep.ladder.module.Board;
+import nextstep.ladder.module.Height;
+import nextstep.ladder.view.InputView;
+import nextstep.ladder.view.ResultView;
+
+public class Main {
+
+    private final static InputView inputView = new InputView();
+    private final static ResultView resultView = new ResultView();
```

Comment:
[컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)에 따라 순서를 지켜주시면 좋을 것 같아요 
```suggestion
    private static final InputView inputView = new InputView();
    private static final ResultView resultView = new ResultView();
```